src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -270,6 +270,22 @@ public TimeSpan Expect100ContinueTimeout
             }
         }
 
+        // Internal for now, until API is approved (#30527)
+        internal Version MaxHttpVersion
+        {
+            get => _settings._maxHttpVersion;
+            set
+            {
+                if (value != HttpVersion.Version11 && value != HttpVersion.Version20)
```

Comment:
Not Version10?

Reply:
Because?  I don't really care, but I'm curious why it wouldn't be allowed.

Reply:
We could support it, but it seems really low-value. Customers that need to restrict to 1.0 can already set the version on the request object. I expect this is very uncommon to do.

Reply:
Ok.  Just seems like an artificial limitation to me.

Reply:
Any plans to allow disabling 1.1 and require 2?